internal/use_cases/create_pull_request.go

```diff
@@ -110,6 +127,11 @@ func (cpr CreatePullRequest) Execute() error {
 			return err
 		}
 
+		// After stablishing the branch, fetch it to get the latest chages.
+		// We don't check the error because the branch may not exist in the
+		// remote repository.
```

Comment:
```suggestion
		// After stablishing the branch, fetch it to get the latest changes.
		// We don't check the error because the branch may not exist in the
		// remote repository.
```